boba-bindings/Makefile

```diff
@@ -0,0 +1,154 @@
+SHELL := /bin/bash
```

Comment:
This has diverged quite a bit from upstream at this point.  Might be worth adopting what's been done there?

Reply:
The Makerfile is synced to op-bindings.